### DISCUSSION TITLE
Noise Figure ngspice

### DISCUSSION THREAD

**Discussion by tomhajjar** [ra3xdh/qucs_s#561] [Q&A]
@dwarning pointed out ngspice provides incorrect NF data for S-parameter analysis of active devices. The future release 43 of ngspice has the fix. Plot below is wrong. I don't have release 43....
 
![2024-02-12_150013](https://github.com/ra3xdh/qucs_s/assets/102838604/ab35e10f-92dd-4d70-bd02-00097ca2dd34)
[VBIC_Analysis_prj.zip](https://github.com/ra3xdh/qucs_s/files/14289216/VBIC_Analysis_prj.zip)

**Comment by ra3xdh** [comment]
This is Ngspice issue not related to Qucs-S GUI itself. Moving to discussions.

**Comment by tomhajjar** [comment]
ngspice nightly build dated 2/13/24 fixes the issue. It is called 42+ but contains updates for 43

https://ngspice.sourceforge.io/download.html#nightly
